draft-ietf-rats-architecture.md

```diff
@@ -643,6 +624,8 @@ plans to support in the TEEP architecture {{-teep-arch}}.
 
 # Roles and Entities
 
```

Comment:
Just make sure we do not merge this PR with both versions (but you knew that).   Right now I think the "Henk version" reads better, and is shorter.